cypress/integration/pages/index.js

```diff
@@ -96,10 +96,94 @@ export const runCommonTests = ({ service, pageType }) => {
           );
         });
 
-        it('should have the correct shared social media metadata', () => {
-          cy.checkSharedSocialmediaMetadata({
-            fbAdmins: '100004154058350',
-            appID: '1609039196070050',
+        it('should have the correct shared metadata', () => {
+          // const description = pageType === 'articles' ? (appConfig[service].promo.summary || appConfig[service].promo.headlines.seoHeadline) : appConfig[service].metadata.summary;
```

Comment:
?

Reply:
Test now fixed and uncommented: https://github.com/bbc/simorgh/pull/3364/files#diff-6ef73c82b1834f188b8648a2ce247f32R130